### DISCUSSION TITLE
Download video shows blank with no audio

### DISCUSSION THREAD

**Discussion by Starlyvil** [aclap-dev/video-downloadhelper#828] [Help and Q&A]
### Your question

After downloading video on udemy using the Video DownloadHelper, the downloaded video flickers when played with no audio. Here is are sample screenshot from 2 different media players:

**PotPlayer:**
![image](https://github.com/aclap-dev/video-downloadhelper/assets/31422222/0f0489d7-cb6c-4da4-a957-f0584d15564e)

**Window Video player:**
![image](https://github.com/aclap-dev/video-downloadhelper/assets/31422222/6b04386b-1bc9-4067-87d7-bddaad905cc1)

And here is the download history from the extension:
![image](https://github.com/aclap-dev/video-downloadhelper/assets/31422222/89e12102-a48a-49a0-90ba-d4e2b5082b5f)

Please help on what could cause this.

Also to note, the video on udemy does not have Download option, I don't know if this could be the cause, but I was able to download the very 1st Introduction video of the course, which also does not have download option enabled.

### Addon & CoApp version and details

version: 9.0.2.3
target: microsoft
channel: stable
lang: en-GB
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36 Edg/125.0.0.0
{
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\nessien\\Downloads"
        }
    },
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 4,
    "successfull_dl": 1
}

**Comment by GrampaWildWilly** [comment]
Search this forum for discussions of DRM.  That's your search key.  DRM.

**Reply by Starlyvil** [reply]
Alright, thank you, I will do that
